internal/k8s/env.go

```diff
@@ -1,11 +1,17 @@
 package k8s
 
 import (
+	"context"
+	"path/filepath"
 	"strings"
 
+	"github.com/mitchellh/go-homedir"
```

Comment:
i used homedir for consistency with k3d, and it's also supposed to be more robust cross-os?
https://github.com/rancher/k3d/blob/e56be1344eeba451285b26934298f8b6e5da7c5e/cli/cluster.go#L85